docs/cpp/configure-intellisense-crosscompilation.md

```diff
@@ -0,0 +1,75 @@
+---
+Order: 9
+Area: cpp
+TOCTitle: Configure IntelliSense for cross-compiling
+ContentId: 381b7ce1-5766-49b0-ad26-f9eedae70e63
+PageTitle: Configure IntelliSense for cross-compilation
+DateApproved: 5/11/2020
+MetaDescription: Configure VS Code c_cpp_properties.json to get IntelliSense when you are compiling for a different platform
+---
+# Configure IntelliSense for cross-compiling
+
+The article is about configuring the C/C++ Extension to provide proper IntelliSense when you compile for a different architecture than your development host machine. Such as when your host machine is x64 but you are compiling for ARM.
+
+The C/C++ extension doesn't provide compilation features itself--it works using a compiler you specify. It does provide IntelliSense. For the extension to provide correct IntelliSense, and to reflect the right sizes of data types, you need to configure the C++ extension to emulate the target architecture.
+
+These configuration settings are stored in your project's `c_cpp_properties.json` file. To edit this file, select **C/C++: Edit Configurations (UI)** from the Command Palette (`kb(workbench.action.showCommands)`):
+
+![Command Palette](images/cpp/command-palette.png)
+
+## Example IntelliSense configuration
+
+ The following shows configuring the C/C++ extension for a Linux x64 host machine that targets Linux ARM.  It configures the following C++ extension settings:
```

Comment:
Should "C++ extension" be "C/C++ extension" or full name of the extension?

Reply:
Or label the settings as "IntelliSense" instead?

"It configures the following IntelliSense settings" which supports the subtitle "Example IntelliSense configuration".

Reply:
+1 to using Intellisense settings. done.